cockroachcloud/cloud-sso-sql.md

```diff
@@ -0,0 +1,173 @@
+- [Authenticating service accounts](#authenticate-application-users-service-accounts) with an external token issuer
+- [Configuring the cluster settings](#configure-your-cluster-to-accept-your-external-identity-provider) required to authenticate service accounts
+
+Note that this regards SQL access to a specific CockroachDB Cluster, not access to a {{ site.data.products.db }} organization. For the latter, see [Single Sign-On (SSO) for {{ site.data.products.db }} organizations](cloud-org-sso.html).
+
+You might also be looking for [Cluster Single Sign-On (SSO) for Self-hosted CockroachDB](../v22.2/sso-sql.html).
+
+## Authenticate human users with the cloud console
+
+{{ site.data.products.db }} can issue authentication tokens for human users, which can be used for SSO using the `ccloud` CLI. {{ site.data.products.dedicated }} and {{ site.data.products.serverless }} clusters are, by default, configured to allow Cluster SSO authentication with {{ site.data.products.db }} serving as SSO provider. No additional set-up is required.
+
+{{site.data.alerts.callout_info}}
+Note that this authentication method only works for human users, since only humans may have {{ site.data.products.db }} Console identities.
+
+Application users (i.e. service accounts), can authenticate using JWT tokens from your own identity provider. See [Authenticate application users (service accounts) with Cluster SSO](#authenticate-application-users-service-accounts).
+{{site.data.alerts.end}}
+
+**Prerequisites:**
+
+- You must have a user identity on a {{ site.data.products.db }} organization. For help setting up an organization and cluster, see: [Quickstart with CockroachDB](quickstart.html).
+- Your {{ site.data.products.db }} user identity must have access to at least one cluster in your organization.
+- To authenticate to a specific cluster using SSO, a {{ site.data.products.db }} user must have a corresponding SQL user already [created](../{{site.versions["stable"]}}/create-user.html#create-a-user) on that cluster. {{ site.data.products.db }} generates a SSO SQL username for each console, corresponding to the user's email by the convention `sso_{email_name}`, where `email_name` is everything up to the `@` in an email address, for example the SQL user `sso_docs` would result from `docs@cockroachlabs.com`. `ccloud` will prompt you to make this user if it does not already exist, in which case an admin must create it manually. 
+- [`ccloud`, the {{ site.data.products.db }} CLI](ccloud-get-started.html) must be installed on your local workstation.
+
+1. First authenticate to your {{ site.data.products.db }} organization. This command will cause your workstation's default browser to open to a {{ site.data.products.db }} authentication portal. Authenticate here as you normally do to the {{ site.data.products.db }} Console. The `ccloud` utility will receive an authentication token from the browser, allowing you to authenticate from the command line.
+
```

Comment:
For the steps here, I would try to make this section align as closely as possible to [this internal guide that we have](https://cockroachlabs.atlassian.net/wiki/spaces/CDX/pages/2821062718/Cloud+Cluster+SSO+Guide).